ocaml/xen-api-client/async_examples/event_test.ml

```diff
@@ -74,7 +74,7 @@ let watch_events rpc session_id =
     Event.from ~rpc ~session_id ~classes:["*"] ~token:"" ~timeout:0.
     >>= fun rpc ->
     let e = event_from_of_rpc rpc in
-    if List.length e.events = 0 then error "Empty list of events" ;
+    if List.is_empty e.events then error "Empty list of events" ;
```

Comment:
is_empty is not available in ocaml 4.14

Reply:
This is not OCaml's stdlib List, it's some other library! That's why it compiled and worked.

Reply:
ah, yeah, like the Hashtbl.find which returns an option